ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/teaser/v1/teaser/teaser.html

```diff
@@ -0,0 +1,32 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2018 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div data-sly-use.teaser="com.adobe.cq.wcm.core.components.models.Teaser"
```

Comment:
Since `CommerceTeaser` extends teaser do we actually need both models here? Would it be easier to only use `data-sly-use.teaser="com.adobe.cq.commerce.core.components.models.teaser.CommerceTeaser"`

Reply:
`CommerceTeaser` only models the actions, thus we need to keep both `Teaser` and `CommerceTeaser` as well, the sightly code uses `CommerceTeaser` to render actions.